docs/api/field_type_reference.md

```diff
@@ -1501,6 +1501,165 @@ new FieldType\Value([
 ]);
 ```
 
+### GraphQL Field Type operations
+
+To get a Field of the Matrix Field Type with GraphQL, you will need to specify a Content ID, a Content Type, and a Field Type. 
+
+The types that are used for input and returned are named after the Type and the Field: 
+
+- `{TypeIdentifier}{FieldIdentifier}RowInput` e.g. `dish.nutritionFacts`, `event.agenda`
+- `{TypeIdentifier}{FieldIdentifier}Row`
+
+The example below shows a GraphQL query for a Recipe Content item (belonging to a Content Type with a Matrix Field added), that has two Fields:
+
+- `name`: `ezstring`
+- `ingredients`: `ezmatrix` with two columns: `ingredient` and `quantity`
+
+```
+{
+  content {
+    recipe(id: 123) {
+      name
+      ingredients {
+        ingredient
+        quantity
+      }
+    }
+  }
+}
+```
+
+The Type returned for the Matrix Field exposes columns defined in the Field definition:
+
+```
+{
+  "data": {
+    "content": {
+      "recipe": {
+        "name": "Cake ingredients",
+        "ingredients": [
+          {
+            "ingredient": "Butter",
+            "quantity": "200 grams"
+          },
+          {
+            "ingredient": "Sugar",
+            "quantity": "100 grams"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+#### Query for the Field Type and Field definition's details
+
+With this query you can inspect:
+
+- the existing types 
+- details of Content Types, and their Fields in the context of developing your own application
+
+In case of a Matrix Field, you can ask for the list of columns, their names and identifiers.
+
+
+```
+{
+  content {
+    _types {
+      recipe {
+        ingredients {
+          settings {
+            minimumRows
+            columns {
+              name
+              identifier
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The response will list the exposed Field Type settings:
+
+- minimumRows
+- columns
+    - name
+    - identifier
+
+Example response:
+
+```
+{
+  "data": {
+    "content": {
+      "_types": {
+        "recipe": {
+          "ingredients": {
+            "settings": {
+              "minimumRows": 1,
+              "columns": [
+                {
+                  "name": "ingredient",
+                  "identifier": "ingredient"
+                },
+                {
+                  "name": "quantity",
+                  "identifier": "quantity"
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+#### Mutation
+
+To create a Matrix Field Type you need to define Field Type and Field definition identifiers.
```

Comment:
Maybe this is where the `RowInput` type from above could be documented, since it is where it is used.